src/store/type.ts

```diff
@@ -977,6 +975,7 @@ export type SettingStoreState = {
   splitTextWhenPaste: SplitTextWhenPasteType;
   splitterPosition: SplitterPosition;
   confirmedTips: ConfirmedTips;
+  engineSettings: EngineSettings;
```

Comment:
好みかもですが、他の〜sは複数の方を型定義してないので、合わせてこうすると揃うかもとか思いました。
```suggestion
  engineSettings: Record<string, EngineSetting>;
```
（こうするとEngineSettingsは不要になりそう）
（まあコメントしてますがかなりどっちでも良いかもです！）

Reply:
ストアで使ってますね